client/src/components/Workflow/WorkflowList.vue

```diff
@@ -202,7 +202,7 @@ export default {
             const id = workflow.id;
             const tags = workflow.tags;
             const data = {
-                show_in_tool_panel: checked,
+                show_in_tool_panel: !checked,
```

Comment:
Is this right?  Don't we want to show it in the tool panel if it is now checked?

I think the conditional below on line 211 needs to be swapped around instead, potentially?

Reply:
> Is this right? Don't we want to show it in the tool panel if it is now checked?

If it's already checked and we received a click on it (a click on filled star), we probably want to hide it, right? 

> I think the conditional below on line 211 needs to be swapped around instead, potentially?

Well, I guess I'd need to swap it at
https://github.com/galaxyproject/galaxy/blob/07a9e0c2b1e075b76da7357c708f945dd3e7a7e8/client/src/components/Workflow/WorkflowList.vue#L62 
is it something, you'd like to see?

Reply:
Ahh, yep, that's where it goes weird.  If it is a solid icon, meaning checked, that should probably return 'false', and the other 'true', as the 'checked' value for clarity?  Since that would indicate the on-event 'checked' model value accurately.